validator/src/main/java/com/amazon/aoc/validators/ContainerInsightStructuredLogValidator.java

```diff
@@ -42,7 +45,10 @@ void init(Context context, FileConfig expectedDataTemplate) throws Exception {
     logGroupName = String.format("/aws/containerinsights/%s/performance",
             context.getCloudWatchContext().getClusterName());
     MustacheHelper mustacheHelper = new MustacheHelper();
-    for (String logType : LOG_TYPE_TO_VALIDATE) {
+    String[] logTypeParse = expectedDataTemplate.getPath().toString().split("/");
```

Comment:
I would recommend against this. What happens if we need another test case that only validates against the Cluster? I think the setup discussed before worked better. Using a try catch to skip the other log types if the files don't exist.

Reply:
Yeah, I believe the try catch method would be a lot better since that would work for every test case; I implemented it like this so people would be able to select what expected metric type they would want since there's a lot of different combinations.